### DISCUSSION TITLE
UI rendering cuts off bottom of colored single row cells in table (7.5.6 + 8b1/2)

### DISCUSSION THREAD

**Discussion by lux4rd0** [grafana/grafana#38283] [Dashboards, Panels & Visualizations]
**What happened**:

When displaying tables that are only a single row height, the bottom part of the row is chopped off. (Using 3 segments in the layout editor: one for the title, one for the row header, and one for the cell).

If you give that panel four-block layout segments, you can see the full row.

**What you expected to happen**:

It would be nice that single-row tables (or multi rows) don't get cut off so that each table row equates to the UI panel segment size. There's actually room in the URL to show more the bottom of the cell - it's just cut off with gray border space. (Notable when using colored cell backgrounds)

**How to reproduce it (as minimally and precisely as possible)**:

<img width="1592" alt="table format" src="https://user-images.githubusercontent.com/30187533/119029820-c1915380-b96e-11eb-8258-833b1409f3e3.png">

**Anything else we need to know?**:

**Environment**:
- Grafana version: Grafana v7.5.6 (5fdad5bfe4) and Grafana v8.0.0-beta2 (9d4c719205)
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser: MacOS - Chrome Version 90.0.4430.212 (Official Build) (x86_64)
- Grafana plugins:
- Others:

**Comment by hugohaggmark** [comment]
Confirmed on Grafana Play https://play.grafana.org/d/b2ZF4Y6Gz/ui-rendering-cuts-off-bottom-of-colored-single-row-cells-in-table-7-5-6-8b1-2-34529?orgId=1

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by antoniy** [comment]
Having the same issue with all my tables.

Standard size:
<img width="1495" alt="image" src="https://user-images.githubusercontent.com/105340080/233830667-1d756871-2478-4b14-a07a-859da1f4bff0.png">

Resized enough to show the content:
<img width="1493" alt="image" src="https://user-images.githubusercontent.com/105340080/233830722-4898fe53-a5b6-49bb-a52e-5c2ced7f98aa.png">

Any progress on that one?

Environment:

Grafana version: [v10.0.0-112644pre (24c754334f)](https://github.com/grafana/grafana/blob/main/CHANGELOG.md)
Data source type & version:
OS Grafana is installed on:
User OS & Browser: MacOS - Brave Version 1.49.120 Chromium: 111.0.5563.64 (Official Build) (arm64)
Grafana plugins:
Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
